eslint.config.mjs

```
@@ -1,22 +1,19 @@
 
-const eslintConfig = defineConfig([
-  ...nextVitals,
-  nextTs,
-  {
-    plugins: {
-      prettier: prettierPlugin,
-    },
-    rules: {
-      "prettier/prettier": "error",
-    },
+const eslintConfig = defineConfig([...nextVitals, nextTs, {
```

ESLint

[error] Replace `...nextVitals,·nextTs,` with `⏎··...nextVitals,⏎··nextTs,⏎·`

(prettier/prettier)